File: backend/jangan-backend/src/main/java/com/ssafy/jangan_backend/beacon/dto/BeaconNotificationDto.java
package com.ssafy.jangan_backend.beacon.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
public class BeaconNotificationDto{
	private String beaconName;
	private String imageUrl;
	private boolean isNewFire;
}

File: mobile/app/src/main/java/com/ssafy/jangan_mobile/service/dto/BeaconNotificationDto.kt
package com.ssafy.jangan_mobile.service.dto

data class BeaconNotificationDto(
    val beaconName: String,
    val beaconCode: Int,
    val coordX: Int,
    val coordY: Int,
    val floor: Int,
    val imageUrl: String,
    val isNewFire: Int
)
